cl-postgres/tests/tests.lisp

```diff
@@ -221,8 +232,8 @@ variables:~:{~%  ~A: ~(~A~), ~:[defaults to \"~A\"~;~:*provided \"~A\"~]~}~%"
 
 (test bulk-writer
   (with-test-connection
-    (exec-query connection "create temporary table test (a int, b text, c date, d timestamp, e int[])")
-    (let ((stream (open-db-writer *test-connection* 'test '(a b c d e))))
+    (ignore-errors (exec-query connection "create table test_bulk_writer (a int, b text, c date, d timestamp, e int[])"))
```

Comment:
`ignore-errors` in tests seems ugly. What kind of errors are signaled in here? It is possibly to explicily `handler-case` them instead?

Reply:
Old sql code that has never been updated. Pre Postgresql 9.1 the sql create table syntax did not have the ability to specify "if not exists". So the ignore-error was just there in case the table already existed and postgresql complained. Will change the sql and eliminate the ignore-erros.